spectacles/cli.py

```diff
@@ -440,6 +442,32 @@ def _build_assert_subparser(
     )
 
 
+def log_failing_sql(
```

Comment:
Maybe this is something we do in a future PR, but it would be _really_ cool to put the error message and LookML link (just like we do in the CLI output) in a comment block at the top of this file.

If it would be useful, we could also add a timestamp for when the query was run.

For example:
```sql
/*
The Google BigQuery Standard SQL database encountered an error while running
this query. Failed to retrieve data - Name numbers_of_orders not found inside
dim_customers at [3:23]

LookML: https://spectacles.looker.com/projects/jaffle_shop/files/views%2Fdim_customers.view.lkml?line=46

Query executed at 2020-04-09 22:24:19T00:00
*/

SELECT
  dim_customers.numbers_of_orders  AS dim_customers_number_of_orders
FROM analytics.fct_orders  AS fct_orders
LEFT JOIN analytics.dim_customers  AS dim_customers ON fct_orders.customer_id = dim_customers.customer_id

WHERE 1 = 2
GROUP BY 1
ORDER BY 1
LIMIT 0
```

Reply:
I love this. I'm going to suggest I make an issue and add it in a subsequent PR?

Reply:
Fine by me!

Reply:
https://github.com/spectacles-ci/spectacles/issues/183